src/System.Drawing.Common/src/System/Drawing/Printing/Duplex.cs

```diff
@@ -8,7 +8,6 @@ namespace System.Drawing.Printing
     /// <devdoc>
     ///    <para>Specifies the printer's duplex setting.</para>
     /// </devdoc>
-    [Serializable]
```

Comment:
An enum can't be partial. Since enums are serializable by default I think we can live with removing the attribute in Mono.